l10n_it_riba/models/account.py

```diff
@@ -58,9 +59,27 @@ def _compute_is_past_due(self):
             if len(invoice.past_due_move_line_ids) != reconciled_past_due:
                 invoice.is_past_due = True
 
+    def _compute_exposition(self):
+        for invoice in self:
+            if invoice.is_riba_payment:
+                today = fields.Date.today()
+                exposition_line_ids = invoice.line_ids.filtered(
+                    lambda l: l.riba
+                    and l.display_type == "payment_term"
+                    and l.date_maturity > today
+                )
+                invoice.exposition = sum(exposition_line_ids.mapped("balance"))
+            else:
+                invoice.exposition = 0.0
+
     riba_credited_ids = fields.One2many(
         "riba.slip", "credit_move_id", "Credited RiBa Slips", readonly=True
     )
+
+    exposition = fields.Float(
```

Comment:
Solo un appunto al volo, `exhibition` in inglese indica esposizione nel senso di mostra (es. evento pubblico come una fiera), non ha a che fare con l'ambito contabile.

Reply:
E' vero, infatti in chiamata un venerdì avevo chiesto che termine era più appropriato per il campo, ma era passato questo.
Sì potrebbe chiamare "risk" o "overdraft"
Dimmi tu

Reply:
> E' vero, infatti in chiamata un venerdì avevo chiesto che termine era più appropriato per il campo, ma era passato questo. Sì potrebbe chiamare "risk" o "overdraft" Dimmi tu

Un "overdraft" si verifica quando viene prelevata una somma superiore a quella presente in un conto.
In questo caso mi pare di capire che il termine si riferisca alle fatture non pagate ma "non" ancora scadute, giusto?
In tal caso potrebbe andar bene un semplice "open amount".

Reply:
@primes2h ho fatto

Reply:
> @primes2h ho fatto

Grazie :+1:

Reply:
> @primes2h ho fatto

@odooNextev Ah, visto ora. 
Andrebbe modificato anche il messaggio di commit https://github.com/OCA/l10n-italy/pull/3984/commits/1aa2e40eb0c6c15c2be719c1c6f60b47a04df305

Reply:
grazie @odooNextev 

mancherebbe ancora questo punto :wink: 
> Trattandosi di un nuovo campo documenterei la sua funzionalità sul README

Hai modo di farlo?